lib/terraforming/resource/iam_group_membership.rb

```diff
@@ -44,11 +44,12 @@ def tfstate
       private
 
       def group_members_of(group)
-        @client.get_group(group_name: group.group_name).users.map { |user| user.user_name }
+        @client.get_group(group_name: group.group_name).map(&:users).flatten.
```

Comment:
__[rubocop]__
* Place the . on the next line, together with the method name. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)